### PR TITLE
LL-2976 Manager + Onboarding Device actions

### PR DESCRIPTION
**Pull request by LFBarreto**
Adapt the manager + onboarding flow to new device actions 
will need a live-common bump as well
### Type

Feature

### Context

LL-2976

### Parts of the app affected / Test plan

Manager + Onboarding flow

### CODE REVIEW

**Comment by gre** [comment]
@LFBarreto the work from Jun is merged. could you try to rebase with only your change on top of develop?

**Review by Arnaud97234 — CHANGES_REQUESTED** [review]
### Bug found
1) iOS: After a successful Onboarding (device paired) LL shouldn't be in read only mode. Can't leave read only mode even when pairing a NanoX after the onboarding.
______
______

2) Android - Manager - Step "Allow manager / device connection": Deny on the device OR disconnect device lead to a loop that user can't leave:
- an error drawer is displayed (connectManagerTimeout OR ManagerRefusedOnDevice OR deviceDisconnected)
- the "X" in the drawer doesn't do any action
- Then the flow restarts automatically without tapping retry button

3) Android - Manager - step "select your device": when a Device is already paired, if I try to connect it and it's OFF I fall in an infinite loop

ezgif com-gif-maker

File: src/screens/Onboarding/steps/pair-new.js [hunk 1]
```diff
@@ -34,10 +43,29 @@ class OnboardingStepPairNew extends Component<OnboardingStepProps> {
 
   pairNew = () => this.props.navigation.navigate(ScreenName.PairDevices);
```
Comment (JunichiSugiura) [inline src/screens/Onboarding/steps/pair-new.js:44]: Is this used still? also `this.help`

**Review by JunichiSugiura — CHANGES_REQUESTED** [review]

**Review by Arnaud97234 — APPROVED** [review]

**Review by gre — APPROVED** [review]

**Comment by gre** [comment]
i'll bump live-common on develop

**Pull request merged**